mkosi/documentation.py

```diff
@@ -32,11 +32,15 @@ def show_docs(
                     logging.warn("pandoc is not available")
                     continue
                 pandoc = run(
-                    ["pandoc", "-t", "man", "-s", resources / f"man/{manual}.{man_chapter}.md"],
+                    [
+                        "pandoc", "-t", "man",
+                        "-s", resources / f"man/{manual}.{man_chapter}.md",
+                        "--lua-filter", resources / "/pandoc/md2man.lua",
```

Comment:
Without this change this does not work
```suggestion
                        "--lua-filter", resources / "pandoc/md2man.lua",
```